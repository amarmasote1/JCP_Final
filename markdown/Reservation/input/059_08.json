{
    "reservation": [
        {
			"storeID": "1322",
            "itemID": "34530001082",
            "uom": "EACH",
            "shippable": "true",
            "procurementAllowed": "false",
            "qty": "4",
            "lineID": "1",
            "reservationID": "CR001-096"
        }
    ]
}