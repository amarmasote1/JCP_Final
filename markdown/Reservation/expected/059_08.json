{
    "message": "Success",
    "body": [
        {
            "partiallyReservedQty": 0,
            "shortageQty": 0,
            "orgCode": "JCP",
            "reservedQty": 4,
            "requestLine": {
                "deliveryMethod": null,
                "channel": null,
                "lineID": "1",
                "storeID": "1322",
                "itemID": "3453000-1082",
                "schedule": false,
                "uom": "EACH",
                "shippable": true,
                "reservationID": "CR001-096",
                "procurementAllowed": false,
                "qty": 4,
                "bopus": false,
                "increaseCapacity": null,
                "shutdown": false
            },
            "problemType": "SUCCESS"
        }
    ],
    "status": 200
}